Collaboration/src/CollaborationServer.js

```diff
@@ -3,6 +3,8 @@ const socketIo = require('socket.io');
 const env = require('./loadEnvironment');
 const logger = require('./Log');
 const { Event } = require('./constants');
+const { getBoilerplate } = require('./utils/helpers');
+const { RedisMemory } = require('./RedisMemory');
 
```

Comment:
Perhaps can change the ordering to 
1. ./Redis
2. ./utils
3. ./constants
4. ./load
5. ./Log